### ISSUE TITLE
cc bcc not working

### ISSUE THREAD

**Issue by Lab-doc** [opto/nostalgy-xpi#72]
"When composing a message, typing cc before the email address, changes the 'to' to 'cc' (or respectively, 'bcc')". 

Doesn't work for me. Newly installed TB78.6, Nostalgy++ 2.2.4

**Comment by hkclark** [comment]
I have noticed this as well.  I have been so thrilled to have my Nostalgy again that I haven't wanted to "complain" and mention it, :-)  but I have noticed that this very nice feature opto added does appear to be broken.  I'm happy to provide more info if it's difficult to recreate for some people.  Thanks!

**Comment by hi5** [comment]
For now esc+c and esc+b works https://github.com/opto/nostalgy-xpi/issues/56

**Comment by Lab-doc** [comment]
Hello @Hi5, 
esc+c and esc+b do not work for me. I have the latest nostalgy++ installed in a fresh TB instance. It is TBx64, in case that is an issue.

**Comment by opto** [comment]
Hello @Lab-doc, 
did you try esc - pause - c? esc and c must be pressed after another within ca. 0.4 seconds (depending on PC).

Klaus

**Comment by Lab-doc** [comment]
Yes. It does not work for me. The "c" remains in the To: field and addresses that start with "c" are offered. It could be that CardBook is conflicting.

**Comment by vanous** [comment]
I use CardBook and Esc-c, Esc-b work, but you have to keep Esc pressed and press c or b with it.

**Comment by Lab-doc** [comment]
>I use CardBook and Esc-c, Esc-b work, but you have to keep Esc pressed and press c or b with it.
Ahh! That works! Thanks.

**Comment by opto** [comment]
@vanous: which TB version/N++ version/OS?
I had to increase the time interval for detection of esc and letter for Win10. Maybe I should make that changeable in an option? Or at least a preference?

**Comment by Lab-doc** [comment]
My opinion: make the time interval long enough that the usual way it works is "ESC" release "b" or "c", without keeping ESC pressed. I do not think an option to alter the timing is needed.

**Comment by opto** [comment]
unfortunately, we do need a preference. If I set it such that it works on my 8 year old notebook, it is unusable on a new PC. Unfortunately, the delay period is called millisecond, but it is different as per processor in your PC.

**Comment by vanous** [comment]
> @vanous: which TB version/N++ version/OS?

Thunderbirg 78.7.0 (64-bit)
N++ 2.2.4
running on Debian bullseye/sid (64-bit)
Eight 2.77GHz Intel i7 Processors

**Comment by Lab-doc** [comment]
@vanous - good point. FWIW, my system is: 
TB 78.7.0 (64-bit)
N++ 2.2.4
Win 10 x64 (20H2)
Lenovo P50 (Xeon E3-1505M @2.80GHz; 64GB RAM; 2x1 TB SSD in RAID 0)
So, a relatively fast system, although now 4 yr old.

**Comment by dmusican** [comment]
This confused me a bunch; I couldn't figure out why ESC-c wasn't working for me. Thanks for the tips above. At the very least, this should be documented clearly in the help pane if this can't be fixed otherwise.
